azurefunctions-extensions-bindings-blob/azurefunctions/extensions/bindings/blob/blobClient.py

```diff
@@ -39,3 +38,16 @@ def get_sdk_type(self):
             )
         else:
             return None
+
+
+def validate_connection_string(connection_string: str) -> str:
+    """
+    Validates the connection string. If the connection string is
+    not an App Setting, an error will be thrown.
+    """
+    if not os.getenv(connection_string):
```

Comment:
also check for empty or blank value?

Reply:
If the connection param isn't provided in the decorator, the function will fail indexing. Added checks for None / empty values